compiler/luci-interpreter/src/kernels/Mean.cpp

```diff
@@ -169,6 +174,9 @@ void Mean::execute() const
     case DataType::U8:
       evalQuantized();
       break;
+    case DataType::S16:
+      evalSInt16();
```

Comment:
I think we need to stick to one naming convention to avoid ambiguity.
I mean this is quantized computations too, so It would be better to name eval functions like
`evalQuantizedU8` and `evalQuantizedS16`

Reply:
Fixed.